app/javascript/packages/document-capture/context/ui-config.tsx

```diff
@@ -0,0 +1,17 @@
+import { createContext } from 'react';
+
+export interface UIConfigContextProps {
+  /**
+   * Specify whether to show the not-ready section on doc capture screen.
+   * Populated from backend configuration
+   */
+  notReadySectionEnabled: boolean;
+}
+
+const UIConfigContext = createContext<UIConfigContextProps>({
```

Comment:
I might worry that "UI Config" could be interpreted as a "miscellaneous" bucket and that this risks becoming a dumping ground for a lot of unrelated configuration. Then again, given how extreme it's become with how many context providers we have these days, maybe that's not a bad thing. I don't fully understand the intended use-case of this feature, but could we stretch to fit it in one of the other existing contexts?

Reply:
@aduth, this is to contain configurations for UI elements(mostly informational and navigational) that may be temporary or staged release. I have looked other contexts at first and not sure which existing context is the best place to put this(also, we have a flag with similar purpose, yes we can consolidate under one json entry for sure), what is your suggestion?

Reply:
I think the new name works 👍 Maybe there'll be some confusion about whether it's better to add a feature flag to this context vs. one of the more feature-specific contexts, but I think that's pretty low stakes.

At some point, maybe it's time to reconsider our approach to "global" state and consider whether some of the other tools in the state management space can help organize this better.

Reply:
@aduth, agree. We have several technologies on the FE, not only system wide feature configurations, also we are passing testing bucket informations in fragments to FE(this can be user session specific).